scripts/plot_skymap.py

```diff
@@ -0,0 +1,55 @@
+from argparse import ArgumentParser
+
+from astropy import units as u
+from gammapy.maps import WcsMap
+from matplotlib import pyplot as plt
+
+parser = ArgumentParser()
+parser.add_argument("-i", "--input-path", required=True)
+parser.add_argument("-o", "--output-path", required=True)
+args = parser.parse_args()
+
+angle = u.deg
+
+
+def main(input_path, output_path):
+    skymap = WcsMap.read(input_path, map_type="wcs")
+
+    geom = skymap.geom
+    source = geom.center_skydir
+
+    edges = u.Quantity(
+        [
+            u.Quantity(geom.pix_to_coord([i, j]))
+            for i, j in zip(range(int(geom.npix[0])), range(int(geom.npix[1])))
+        ],
+    ).T
+
+    fig, ax = plt.subplots()
+
+    ax.pcolormesh(
+        *edges.to_value(angle),
+        skymap.data[0, ...],
+        rasterized=True,
+    )
+
+    ax.scatter(
+        source.ra.to_value(u.deg),
+        source.dec.to_value(u.deg),
+        ec="k",
```

Comment:
should these be set via the matplotlibrc?

Reply:
I would say no: this is a single marker, that needs to be distinguished from the background (colormap). White with black edges is quite distinguishable on all backgrounds (either the white face *or* the black edge make the contrast).